XenAdmin/Wizards/GenericPages/SelectMultipleVMDestinationPage.cs

```diff
@@ -401,12 +406,25 @@ private void PopulateDataGridView(List<ReasoningFilter> homeserverFilters)
                             }
                         }
 
+                        var items = new List<DelayLoadingOptionComboBoxItem>();
                         foreach (var host in Connection.Cache.Hosts)
                         {
                             var item = new DelayLoadingOptionComboBoxItem(host, homeserverFilters);
-                            item.LoadAndWait();
+                            item.ReasonUpdated +=

```

Comment:
1) Can we rename this event to ReasonChanged, because its handler is DelayLoadedComboBoxItem_ReasonChanged and the firing method is OnReasonChanged and it's both pointless and confusing to have a different name for the event.
2) Where is this event unregistered?

Reply:
1. Will do
2. I missed this and it'll need some work to pass through the combobox - I'll need to think about this more carefully.